packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -102,6 +103,11 @@ If you want to reinitialize Shorebird, please run "shorebird init --force".''');
       } else {
         appId = (await createApp(appName: displayName)).id;
       }
+    } on CodePushForbiddenException {
+      logger.err('''
+A paid account is required to create apps.
```

Comment:
Isn’t a free account also able to create apps now?

Reply:
Ah crap, you're right

Reply:
Fixed